ceph-installer-tests/build/build

```diff
@@ -6,4 +6,7 @@ WORKDIR=$(mktemp -td tox.XXXXXXXXXX)
 
 cd $WORKSPACE/tests/functional
 
+clear_libvirt_networks()
+restart_libvirt_services()
```

Comment:
unless my BASH is terribly wrong (which can certainly be the case) I **think** that these functions don't need to have `()` otherwise it will get you a syntax error.